### PR TITLE
LPS-82202

### PR DESCRIPTION
**Pull request by holatuwol**
https://issues.liferay.com/browse/LPS-82202

### CODE REVIEW

**Comment by holatuwol** [comment]
ci:test:relevant

**Comment by achaparro** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: portal-impl/src/com/liferay/portal/upgrade/v7_0_0/UpgradePortalPreferences.java [hunk 1]
```diff
@@ -83,9 +90,29 @@ protected void upgradeStagingPortalPreferences() throws Exception {
...
+							" contains invalid XML, resetting to default"));
+
+					newPreferences = PortletConstants.DEFAULT_PREFERENCES;
+				}
+
+				if (Objects.equals(oldPreferences, newPreferences)) {
+					continue;
+				}
+
+				ps2.setClob(1, new UnsyncStringReader(newPreferences));
```
Comment (achaparro) [inline portal-impl/src/com/liferay/portal/upgrade/v7_0_0/UpgradePortalPreferences.java]: Hey @holatuwol,

In some databases such MySQL we define preferences as a TEXT field. I think we should keep it as setString, is it needed to change to setClob for any reason?

Thanks.

**Comment by achaparro** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/60481. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by achaparro** [comment]
Thanks @holatuwol !

**Pull request closed without merging**